agate-rest/src/main/java/org/obiba/agate/web/rest/user/UserProfileResource.java

```diff
@@ -36,13 +33,14 @@
   private UserService userService;
 
   @GET
-  public AuthDtos.SubjectDto getProfile(@PathParam("username") String username,
+  public AuthDtos.SubjectDto getProfile(
+    @PathParam("username") String username,
+    @QueryParam("locale") @DefaultValue("en") String locale,
```

Comment:
I modify code, I remove default values, a default translation is already selected (when we check if file exists). So I will let this sample of code select the default language.